tests/image/detection/test_data.py

```diff
@@ -5,11 +5,10 @@
 import pytest
 
 from flash.core.data.data_source import DefaultDataKeys
-from flash.core.utilities.imports import _FIFTYONE_AVAILABLE, _IMAGE_AVAILABLE
+from flash.core.utilities.imports import _FIFTYONE_AVAILABLE, _PIL_AVAILABLE
 from flash.image.detection.data import ObjectDetectionData
-from tests.helpers.utils import _IMAGE_TESTING
```

Comment:
I think this line is needed for tests to pass

Reply:
Didn't realize I removed the line. Thank you.

Reply:
No problem :smiley: